app/src/main/AndroidManifest.xml

```diff
@@ -136,46 +141,46 @@
         android:launchMode="singleTop">
       <!-- TODO -->
       <intent-filter>
-        <action android:name="android.intent.action.GET_CONTENT"/>
+        <action android:name="android.intent.action.GET_CONTENT" />
 
-        <category android:name="android.intent.category.DEFAULT"/>
-        <category android:name="android.intent.category.OPENABLE"/>
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.OPENABLE" />
 
-        <data android:mimeType="*/*"/>
+        <data android:mimeType="*/*" />
       </intent-filter>
       <intent-filter>
-        <action android:name="org.kiwix.kiwixmobile.utils.KiwixSearchWidget.TEXT_CLICKED"/>
-        <action android:name="org.kiwix.kiwixmobile.utils.KiwixSearchWidget.ICON_CLICKED"/>
-        <action android:name="org.kiwix.kiwixmobile.utils.KiwixSearchWidget.MIC_CLICKED"/>
-        <action android:name="org.kiwix.kiwixmobile.utils.KiwixSearchWidget.STAR_CLICKED"/>
+        <action android:name="org.kiwix.kiwixmobile.utils.KiwixSearchWidget.TEXT_CLICKED" />
+        <action android:name="org.kiwix.kiwixmobile.utils.KiwixSearchWidget.ICON_CLICKED" />
+        <action android:name="org.kiwix.kiwixmobile.utils.KiwixSearchWidget.MIC_CLICKED" />
+        <action android:name="org.kiwix.kiwixmobile.utils.KiwixSearchWidget.STAR_CLICKED" />
       </intent-filter>
     </activity>
-    <activity android:name=".settings.KiwixSettingsActivity"/>
-    <activity android:name=".search.SearchActivity"/>
-    <activity android:name=".bookmark.BookmarksActivity"/>
+    <activity android:name=".settings.KiwixSettingsActivity" />
+    <activity android:name=".search.SearchActivity" />
+    <activity android:name=".bookmark.BookmarksActivity" />
 
     <provider
         android:name=".data.ZimContentProvider"
         android:authorities="${applicationId}.zim.base"
-        android:exported="true"/>
+        android:exported="true" />
```

Comment:
Are most of the changes to this file necessary?

Reply:
The extra spaces might be a result of my IDE settings or because the branch was originally based on master - [Corrected](https://github.com/kiwix/kiwix-android/pull/1276/commits/515dfcab577533fbe445a33372265478acd3c36b)